packages/x-components/src/x-modules/scroll/components/window-scroll.vue

```diff
@@ -0,0 +1,227 @@
+<script lang="ts">
+  import { mixins } from 'vue-class-component';
+  import { Component, Prop } from 'vue-property-decorator';
+  import { ScrollDirection, ScrollMixin, xComponentMixin } from '../../../components';
+  import { WireMetadata } from '../../../wiring';
+  import { scrollXModule } from '../x-module';
+
+  type ScrollableTag = 'html' | 'body';
+
+  /**
+   * Main scroll component that depending on the user interactivity emits different events for
+   * knowing when the user scrolls, the direction of scroll and if user reaches the start or end.
+   * All this about main scroll.
+   *
+   * @public
+   */
+  @Component({
+    mixins: [xComponentMixin(scrollXModule)]
+  })
+  export default class WindowScroll extends mixins(ScrollMixin) {
```

Comment:
Why is this component called `WindowScroll` instead of `MainScroll` if the documentation says Main scroll component? just wondering 😅

Reply:
Because the documentation is written in chino mandarin and it wasn't affected by the global replace I did. Fixed it. Thanks